core/network/types/block_request.hpp

```diff
@@ -14,6 +14,8 @@
 #include "primitives/block_id.hpp"
 #include "primitives/common.hpp"
 
+using namespace kagome::common;
```

Comment:
Don't think it's a good idea in a header.

Reply:
Surely it isn't. Will fix.